src/test/java/org/breedinginsight/brapps/importer/ExperimentFileImportTest.java

```diff
@@ -603,7 +603,7 @@ public void verifyFailureNewOuExistingEnv(boolean commit) {
         JsonObject result = JsonParser.parseString(upload.body()).getAsJsonObject().getAsJsonObject("result");
         assertEquals(422, result.getAsJsonObject("progress").get("statuscode").getAsInt(), "Returned data: " + result);
 
-        assertTrue(result.getAsJsonObject("progress").get("message").getAsString().startsWith("Experiment Units are missing Observation Unit Id."));
+        assertTrue(result.getAsJsonObject("progress").get("message").getAsString().startsWith("Error(s) detected in file."));
```

Comment:
```suggestion
        assertTrue(result.getAsJsonObject("progress").get("message").getAsString().startsWith("Experimental entities are missing ObsUnitIDs."));
```
This is to account for the change I suggested in ExperimentProcessor.java.